### ISSUE TITLE
How to check if a queue already exists or not ?

### ISSUE THREAD

**Issue by m2omou** [postwait/node-amqp#401]
Is it possible, using the nodejs rabbitmq client to know if a queue already exist before trying to create it ?

Thanks

**Comment by PritamUpadhyay** [comment]
Please read  **README** [here](https://github.com/postwait/node-amqp#connectionqueuename-options-opencallback)

<h3> connection.queue(name[, options][, openCallback]) </h3> 
`passive`: boolean, default false. If set, the server will not create the queue. The client can use this to check whether a queue exists without modifying the server state.

**Comment by m2omou** [comment]
Yes I have read the readme, but that doesn't really help me, do you have any piece of code to explain it ? 

That would be helpful thank you
